src/Types/Transaction.purs

```diff
@@ -89,17 +122,17 @@ instance semigroupTxBody :: Semigroup TxBody where
     , ttl: lift2 lowerbound txB.ttl txB'.ttl
     , certs: lift2 union txB.certs txB'.certs
     , withdrawals: lift2 appendMap txB.withdrawals txB'.withdrawals
-    , update: txB.update <<>> txB'.update
-    , auxiliary_data_hash: txB.auxiliary_data_hash <<>> txB'.auxiliary_data_hash
+    , update: txB.update </> txB'.update
+    , auxiliary_data_hash: txB.auxiliary_data_hash </> txB'.auxiliary_data_hash
     , validity_start_interval:
         lift2 lowerbound
           txB.validity_start_interval
           txB'.validity_start_interval
     , mint: txB.mint <> txB'.mint
-    , script_data_hash: txB.script_data_hash <<>> txB'.script_data_hash
+    , script_data_hash: txB.script_data_hash </> txB'.script_data_hash
```

Comment:
Potentially a bit confusing but the old `<<>>` use to be `Last` also, I'm just renaming to something a bit nicer so that
`<\>` is `First`
`</> is `Last` (use to be  `<<>>`)
`<<>>` is the new `maybeArrayMerge`. 

Just wanted to make sure it was clear the logic is unchanged.

Secondly, are you wondering why we don't use `maybeArrayMerge`? I don't think it matters too much because we only want to append to `script_data_hash` via an effectful call. In particular, see 
`setScriptDataHash :: Array Redeemer -> Array Datum -> Transaction -> Effect Transaction`. Furthermore, I'm not sure if it makes much sense to mix two transaction `script_data_hash`es together, and this may hold for a few other fields, hence the use of `Last`.

If there is a particular reason we require merging, we can always switch.

Reply:
I like the naming changes a lot. I think I just don't grasp merging tx's together and how can some data from the first one discarded.
Also I don't like the effectful interface, but I guess we can address this in the long run.

Reply:
I actually agree about merging tx. I think we were trying to replicate Plutus `tx`'s `Monoid` instance which may be deprecated now, and in the process, came across fields that were monoidally unintuitive in our `TxBody`. Interestingly `TxInfo` doesn't have a `Semigroup` instance. Perhaps it makes more sense to define an "empty" value, without semigroup or monoid and leave it at that.